cli/tests/e2e/snapshots/test_baseline/test_staged_changes/error.txt

```diff
@@ -0,0 +1,29 @@
+               
+               
+┌─────────────┐
+│ Scan Status │
+└─────────────┘
+  Scanning 1 file tracked by git with 1 Code rule:
+  Scanning 1 file.
+  Current version has 2 findings.
+
+Switching repository to baseline commit '4b63ca493df05e223404a80ac5f30d2aa830a704'.
+  Will report findings introduced by these commits (may be incomplete for shallow checkouts):
```

Comment:
This text makes me think that we're going to report findings introduced by the baseline commit rather than subtract it out. It also makes me think that we're going to leave their repo in the baseline commit stage, which would annoy me

Reply:
Good point, I never updated these printouts, I'll do that so it accurately reflects what's happening, which is creating a separate work tree, not checking out or switching anything